libs/src/services/discoveryProvider/DiscoveryProviderSelection.js

```diff
@@ -117,7 +117,17 @@ class DiscoveryProviderSelection extends ServiceSelection {
     if (status !== 200) return false
     if (service !== DISCOVERY_SERVICE_NAME) return false
     if (!semver.valid(version)) return false
-    if (!this.ethContracts.isValidSPVersion(this.currentVersion, version)) return false
+
+    if (!this.ethContracts.isAheadByPatchVersions(this.currentVersion, version)) {
```

Comment:
can we add this as a comment?
```
1. allow a discprov that is “ahead” by a patch version
2. add a discprov to the backups if it is behind by a patch version
3. reject everything else
```

Reply:
also i think these method names are a little confusing?

Reply:
:)